unit_tests/cmake/test_config/irods_client_server_negotiation.cmake

```diff
@@ -4,7 +4,8 @@ set(IRODS_TEST_SOURCE_FILES ${CMAKE_CURRENT_SOURCE_DIR}/src/main.cpp
                             ${CMAKE_CURRENT_SOURCE_DIR}/src/test_client_server_negotiation.cpp)
 
 set(IRODS_TEST_INCLUDE_PATH ${IRODS_EXTERNALS_FULLPATH_BOOST}/include
-                            ${IRODS_EXTERNALS_FULLPATH_FMT}/include)
+                            ${IRODS_EXTERNALS_FULLPATH_FMT}/include
+                            ${IRODS_EXTERNALS_FULLPATH_JSON}/include)
```

Comment:
To keep inline with what @SwooshyCueb has been doing, you're going to want to replace the JSON line with `nlohmann_json::nlohmann_json`.

Reply:
Done

Reply:
i see your 'Done', but ... it isn't?

Reply:
Sorry, haven't pushed it yet